packages/core/echo/echo-pipeline/src/filter/filter-match.ts

```diff
@@ -148,6 +154,18 @@ export const filterMatchObjectJSON = (filter: QueryAST.Filter, obj: ObjectJSON):
       return true;
     }
 
+    case 'tag': {
```

Comment:
I missed the original change:

```ts
/**
 * Tag.
 */
const FilterTag_ = Schema.Struct({
  type: Schema.Literal('tag'),
  tag: Schema.String, // TODO(burdon): Make OR-collection?
});
export interface FilterTag extends Schema.Schema.Type<typeof FilterTag_> {}
export const FilterTag: Schema.Schema<FilterTag> = FilterTag_;
```

Are you going back on what we've discussed before of doing tags via relations?

Reply:
@dmaretskyi I think this is meant as a stopgap to be refactored to make tags first-class entities with relations in a follow-up.